src/components/editor/editor-screen-table.tsx

```diff
@@ -0,0 +1,374 @@
+   */
+  const buildProductionVersion = (id: string): MetaformVersionRow => {
+    return {
+      id: id,
+      typeString: strings.editorScreen.formProductionVersion
+    };
+  };
+
+  /**
+   * Builds MetaformVersionRows
+   * 
+   * @param metaform metaform
+   */
+  const buildVersionRows = (metaform: Metaform): MetaformVersionRow[] => {
+    const versions = metaformVersions.filter(x => x.data.id as unknown === metaform.id);
+    const productionVersion = buildProductionVersion(metaform.id!);
+    const versionRows: MetaformVersionRow[] = versions.map((version: MetaformVersion) => {
+      return {
+        id: version.id!,
+        typeString: version.type === MetaformVersionType.Archived ? strings.editorScreen.formVersionArchived : strings.editorScreen.formVersionDraft,
+        type: version.type,
+        createdAt: version.createdAt!,
+        modifiedAt: version.modifiedAt!,
+        modifierId: version.lastModifierId!
+      };
+    });
+    versionRows.push(productionVersion);
+
+    return versionRows;
+  };
+
+  useEffect(() => {
+    /**
+     * Handles correct link to editor for edit button
+     */
+    const handleEditorLink = async () => !!selectedId && setLinkToEditor(await getPathToEditor(selectedId));
+
+    if (selectedId) {
+      handleEditorLink();
+    }
+  }, [selectedId]);
+
+  /**
+   * Renders MetaformVersions Listing
+   * 
+   * @param metaform metaform
+   */
+  const renderVersionRows = (metaform: Metaform) => {
+    const versions = buildVersionRows(metaform);
+    
+    if (!versions) {
+      return;
+    }
+
+    return versions.map((version: MetaformVersionRow) => {
+      return (
+        <ListItem
+          key={ version.id }
+          sx={{
+            width: "100%",
+            padding: 0,
+            height: "60px"
+          }}
+        >
+          <AdminFormListStack direction="row">
+            <Grid container>
+              <Grid item md={ 8 }>
+                <AdminFormTypographyField>
+                  <ListRounded style={{ fill: "darkgrey", marginRight: "0.5rem" }}/>
+                  { version.typeString }
+                </AdminFormTypographyField>
+              </Grid>
+              <Grid item md={ 1 }>
+                <AdminFormTypographyField>
+                  <DateRangeRounded style={{ fill: "darkgrey", marginRight: "0.5rem" }}/>
+                  { moment(version.createdAt).format("LLL") }
+                </AdminFormTypographyField>
+              </Grid>
+              <Grid item md={ 1 }>
+                <AdminFormTypographyField>
+                  <DateRangeRounded style={{ fill: "darkgrey", marginRight: "0.5rem" }}/>
+                  { moment(version.modifiedAt).format("LLL") }
+                </AdminFormTypographyField>
+              </Grid>
+              <Grid item md={ 1 }>
+                <AdminFormTypographyField>
+                  <PersonRounded style={{ fill: "darkgrey", marginRight: "0.5rem" }}/>
+                  placeholder_email
```

Comment:
If this is not yet finished feature, it should say "Not yet implemented" etc. in the UI

Reply:
Good idea. Done.